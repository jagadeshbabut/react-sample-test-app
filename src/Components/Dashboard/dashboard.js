import React, {Component} from 'react';
import {Link} from 'react-router-dom';
import './dashboard.css';

class Dashboard extends Component {
    render() {
        return (
            <div className="App">
                <header className="App-header">
                    <Link to="/todos">Go to Todos!</Link>
                </header>
            </div>
        );
    }
}

export default Dashboard;
