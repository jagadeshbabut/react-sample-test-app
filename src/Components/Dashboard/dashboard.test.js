import React from 'react';
import {shallow} from 'enzyme';
import Dashboard from './dashboard';

describe('Dashboard Component', () => {
    it('Renders without crashing', () => {
        shallow(<Dashboard/>)
    });

    it('Dashboard component has not changed', () => {
        const wrapper = shallow(<Dashboard/>);
        expect(wrapper).toMatchSnapshot()
    });

    it('Check Goto Todos link is working', () => {
        const wrapper = shallow(<Dashboard/>);
        // expect(wrapper).
    });

});