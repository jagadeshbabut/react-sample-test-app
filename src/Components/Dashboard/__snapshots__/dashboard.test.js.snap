// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Component Dashboard component has not changed 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <Link
      replace={false}
      to="/todos"
    >
      Go to Todos!
    </Link>
  </header>
</div>
`;
